standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -338,6 +338,9 @@ private ControllerServiceMain startLocalController(int controllerId) {
         RESTServerConfig restServerConfig = RESTServerConfigImpl.builder()
                 .host("0.0.0.0")
                 .port(this.restServerPort)
+                .tlsEnabled(this.enableTls)
+                .keyFilePath("../config/bookie.keystore.jks")
+                .keyFilePasswordPath("../config/bookie.keystore.jks.passwd")
```

Comment:
Please don't apply default paths to the in-proc server!

Reply:
Fixed.